addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -239,6 +242,7 @@ class ChatWindow extends Component {
             this.chatWindow.unfold();
             this.chatWindow.focus();
         } else {
+            this._saveThreadScrollTop();
```

Comment:
Adding a condition on thread/threadViewer here and removing condition on _saveThreadScrollTop looks possible.